### ISSUE TITLE
Creating private issues for student data archival requests!

### ISSUE THREAD

**Issue by balajialg** [berkeley-dsep-infra/hubs#2866]
# Summary

Currently, our [data archival requests](https://github.com/berkeley-dsep-infra/datahub/issues/new?assignees=felder&labels=support&template=data_archival_request.yml) are out to the public. It may not be an ideal scenario as all the student email details are publicly accessible which may not align with FERPA requirements. We should explore creating private issues for such requests so that student details are held privately. 

My initial exploration led me to find this [github issue](https://github.com/isaacs/github/issues/37) to allow private issues in a public repository. Despite this being a feature [requested](https://twitter.com/adam_baldwin/status/385389448965664768) way back in 2013, Github has not solved it, unfortunately (at least based on my current understanding). So creating this issue to explore alternate scenarios where such requests details are held private by using other suggested options such as GitLab or creating a private repository to handle this issue from now on.

# Tasks to complete

1. Explore alternatives to keep student data private
2. Implement the feasible solution

**Comment by balajialg** [comment]
@felder's suggestion is to move such requests to a ticketing system of choice - Service Now or JIRA. To be discussed during the November sprint planning meeting.

**Comment by yuvipanda** [comment]
Yeah, this was the reason I had initially used the email address in the note.

**Comment by flawnn** [comment]
A bit off topic:
But I created a thread in GitHub Discussions to get that feature implemented.
https://github.com/github/feedback/discussions/7971

**Comment by balajialg** [comment]
Thanks a lot, @flawnn for starting the discussion again. Hoping that GitHub prioritizes this feature soon. 

Thanks for giving us a heads-up! Appreciate it.

**Comment by felder** [comment]
@flawnn thanks!  I wonder if github even has any interest in solving this issue though.  "Privacy" is something github charges for, although I guess I could see them including this feature in paid plans.

We also do have plans to automate this in the future so that students can securely obtain their files and they don't have to wait for support to do the retrieval.
